SWLOR.Game.Server/Feature/RecipeDefinition/EngineeringRecipeDefinition/ModuleRecipes.cs

```diff
@@ -494,6 +602,60 @@ private void Tier3()
 				.Component("ref_idailia", 5)
 				.Component("elec_good", 3);
 
+			// Hypermatter Injector III
```

Comment:
I need these recipes added to the Bible too.

Reply:
I'll get a full list of updated stats and recipes your way soon. If you'd like it sooner than later I can get on that otherwise I was planning on waiting until wider testing was underway and further changes are unlikely.